Various things

nseam opened the pull request:
Snapshot of current module code.
- Some added fields (via Field Api) are unrelated to the current entites architecture and need to be modified/removed.
- I didn't ran CodeSniffer yet
- There may be some missing comments
- There are many entity hook function stubs, they may be needed later

nseam committed:
Snapshot of current module code

Snapshot of current module code. Added little bit of documentation for a place to suggestions

Snapshot of current module code

Snapshot of current module code

Snapshot of current module code

nseam committed:
#28 Regression module
Snapshot of current module code

nseam committed:
#28 Regression module
Fixed diffing to be dynamic,
Added removing and switching on/off menu paths,
Implemented fragmented diffing,
Better user experience, more data displayed in Regression Status page,
Fixed regression resolving validation

kenorb commented on template.txt:
```diff
@@ -0,0 +1,10 @@
+<h2>Welcome,</h2>
```
Is this file is used anywhere?

kenorb commented on regression.module:
```diff
@@ -50,33 +59,614 @@ function regression_init() {
+  // Newlines become a "display:block".
+  $body              = str_replace("\n", '<span style="display:block;"></span>', $body);
+  
+  file_put_contents('C:\body.htm', $body);
```
please remove the hardcoded path

nseam replied:
W dniu 2013-04-23 13:29, Rafal pisze:

> In regression.module:
> 
> > -  $revision_new_date = date('Y-m-d H:i:s', $new_revision->updated);
> > -  $revision_new_diff = regression_diff_revisions($old_revision, $new_revision);
> >   +
> > -  $settings          = regression_get_path_configuration($entry->path, TRUE);
> >   +
> > -  $recipients        = $settings['notifications_email_recipients'];
> > -  $template          = $settings['notifications_email_regression_template'];
> >   +
> > -  $css_body          = 'padding: 12px;';
> >   +
> > -  $body              = '<div style="' . $css_body . '">' . regression_replacement_patterns_replace($new_revision, $template) . '</div>';
> >   +
> > -  // Newlines become a "display:block".
> > -  $body              = str_replace("\n", '<span style="display:block;"></span>', $body);
> >   +
> > -  file_put_contents('C:\body.htm', $body);
> 
> please remove the hardcoded path
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/kenorb/regression/pull/1/files#r3913732.
> 
> Done

## 

/Kind regards / Pozdrawiam,
Adrian Kierzkowski/

nseam committed:
#28 Regression module
Removed unneeded code

kenorb commented on includes/regression.admin.paths.edit.inc:
```diff
@@ -0,0 +1,341 @@
+    }
+  }
+
+  if (0)
```
if (0)?

kenorb commented on libraries/finediff/finediff.php:
```diff
@@ -0,0 +1,687 @@
+<?php
```
finediff needs to be installed separately into libraries dir, as the licence is not compatible and added the info in Reports that it's missing

kenorb commented on regression.info:
```diff
@@ -1,5 +1,15 @@
+dependencies[] = ctools
+dependencies[] = html2txt
+dependencies[] = mailsystem
+dependencies[] = mimemail
```
make this dependency optional (enhances[] = mimemail)

kenorb commented on regression.info:
```diff
@@ -1,5 +1,15 @@
+project = regression
+dependencies[] = ctools
+dependencies[] = html2txt
+dependencies[] = mailsystem
```
where this dependency is used?

nseam committed:
#28 Regression module
Fixed diffing to be dynamic,
Added removing and switching on/off menu paths,
Implemented fragmented diffing,
Better user experience,
more data displayed in Regression Status page,
Fixed regression resolving validation

nseam committed:
#28 Regression module
Added content ignoring via RegEx expressions

kenorb commented:
Please sort out the comments.

nseam committed:
ADS-28
FineDiff library now are treated as the external library that Regression depends on,
Added more comments,
Fixed code syntax to be coder_review-friendly
